### ISSUE TITLE
Q key will not release items in your hand

### ISSUE THREAD

**Issue by BlueMints000** [GriefPrevention/GriefPrevention#1942]
### Observed Behavior

As soon as you install the plugin, and you press Q to let go of something, the item stays in the hotbar, you can't let go of items. Also when you try and break a block you get this message "Cannot build unless you make a claim. type /ignoreclaim to ignore this message." 

### Expected Behavior

That the Q button will release the item in my hand.
That breaking a block will not give me annoying messages that I can't build yet until I make a claim.

### Reproduction steps

N/A

### Server version

```shell
Java 1.19.2
```

### GriefPrevention version

```shell
Grief Prevention 16.18
```

### Configuration

```yaml
N/A
```

### Plugin list

```shell
N/A
```

### Running without GriefPrevention

- [X] I attempted running the server without GriefPrevention installed.
- [X] The problem does _not_ occur when GriefPrevention is removed from the server.

### Running with only GriefPrevention

- [ ] I attempted running only GriefPrevention on the server.
- [ ] The issue still occurs when GriefPrevention is the only plugin running.

### Running on a fresh, clean server installation

- [ ] I attempted testing for the issue on a new server.
- [ ] The issue still occurs on a new server.

### Using unmodified client

- [ ] I attempted testing for the issue with the vanilla client.
- [ ] The issue still occurs when using the vanilla client.

### We appreciate you taking the time to fill out a bug report!

- [X] I searched for similar issues before submitting this bug report.

**Comment by bigpresh** [comment]
Can you post your config?  It sounds likely that you've set the `Claims.Mode` setting to `Creative`, or `SurvivalRequiringClaims`, both of which require you to have claimed land before you can build on it - and creative mode rules also prevent dropping items (so that a player can't just keep getting more and more items from their Creative inventory and dropping them everywhere for low-effort griefing).

In other words, it sounds like the plugin is actually working as designed, assuming that you do have `Claims.Mode` set to `Creative` or `SurvivalRequiringClaims` - the [creative claims mode documentation](https://docs.griefprevention.com/configuration/#creative-claims-mode) explains this all pretty well.

**Comment by RoboMWM** [comment]
That's exactly what it is, creative claims mode.

I do intend on removing this mode due to the complexity of Creative mode grief - would be better served in an addon and avoid lesser-known design decisions being implemented as default without user knowledge, such as this ticket.
